// @esri/calcite-components/dist/esm/bs-94234e13.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.8.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-403f5a6f.js';
import './index-ae8503a2.js';
import './form-20239525.js';
import './dom-32bdb50b.js';
import './guid-2a8131ea.js';
import './resources-dda3f237.js';
import './interactive-c790e261.js';
import './browser-689f2af5.js';
import './key-d353e9d1.js';
import './label-cdf7ee61.js';
import './loadable-d2f19cec.js';
import './locale-2c050f01.js';
import './observers-88ad66e4.js';
import './focusTrapComponent-c6e1aaff.js';
import './math-f364afab.js';
import './t9n-ae6e1b73.js';

// Bosnian [bs]
var locale = {
  name: 'bs',
  weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
  months: 'januar_februar_mart_april_maj_juni_juli_august_septembar_oktobar_novembar_decembar'.split('_'),
  weekStart: 1,
  weekdaysShort: 'ned._pon._uto._sri._čet._pet._sub.'.split('_'),
  monthsShort: 'jan._feb._mar._apr._maj._jun._jul._aug._sep._okt._nov._dec.'.split('_'),
  weekdaysMin: 'ne_po_ut_sr_če_pe_su'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD.MM.YYYY',
    LL: 'D. MMMM YYYY',
    LLL: 'D. MMMM YYYY H:mm',
    LLLL: 'dddd, D. MMMM YYYY H:mm'
  }
};
dayjs.locale(locale, null, true);

export default locale;
