// @esri/calcite-components/dist/esm/mk-317e7de2.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/main/LICENSE.md for details.
 * v1.8.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-403f5a6f.js';
import './index-ae8503a2.js';
import './form-20239525.js';
import './dom-32bdb50b.js';
import './guid-2a8131ea.js';
import './resources-dda3f237.js';
import './interactive-c790e261.js';
import './browser-689f2af5.js';
import './key-d353e9d1.js';
import './label-cdf7ee61.js';
import './loadable-d2f19cec.js';
import './locale-2c050f01.js';
import './observers-88ad66e4.js';
import './focusTrapComponent-c6e1aaff.js';
import './math-f364afab.js';
import './t9n-ae6e1b73.js';

// Macedonian [mk]
var locale = {
  name: 'mk',
  weekdays: 'недела_понеделник_вторник_среда_четврток_петок_сабота'.split('_'),
  months: 'јануари_февруари_март_април_мај_јуни_јули_август_септември_октомври_ноември_декември'.split('_'),
  weekStart: 1,
  weekdaysShort: 'нед_пон_вто_сре_чет_пет_саб'.split('_'),
  monthsShort: 'јан_фев_мар_апр_мај_јун_јул_авг_сеп_окт_ное_дек'.split('_'),
  weekdaysMin: 'нe_пo_вт_ср_че_пе_сa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'D.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY H:mm',
    LLLL: 'dddd, D MMMM YYYY H:mm'
  },
  relativeTime: {
    future: 'после %s',
    past: 'пред %s',
    s: 'неколку секунди',
    m: 'минута',
    mm: '%d минути',
    h: 'час',
    hh: '%d часа',
    d: 'ден',
    dd: '%d дена',
    M: 'месец',
    MM: '%d месеци',
    y: 'година',
    yy: '%d години'
  }
};
dayjs.locale(locale, null, true);

export default locale;
